PortalFetch/crawler_sel.py

```diff
@@ -225,22 +216,25 @@ def waitUtilPageLoaded(driver):
     Parameters: webdriver
     Returns: None
     """
-    while True:
+    count = 0
+    while True and count < 30:
         try:
             driver.find_element_by_class_name("banner_copyright")
```

Comment:
if the statement `driver.find_element_by_class_name("banner_copyright")` causes an exception, the `except` clause kicks in, so `count` would not increment if "banner_copyright" is not found.